docs/concepts/memap.md

```diff
@@ -64,20 +66,34 @@ The table above showed multiple memory sections.
 - ``.text``: is where the code application and constants are located in Flash.
 - ``.data``: nonzero initialized variables; allocated in both RAM and Flash memory (variables are copied from Flash to RAM at runtime).
 - ``.bss``: uninitialized data allocated in RAM, or variables initialized to zero.
-- ``Heap``: dynamic allocations in the Heap area in RAM (for example, used by ``malloc``). The maximum size value may be defined at build time.
-- ``Stack``: dynamic allocations in the Stack area in RAM (for example, used to store local data, temporary data when branching to a subroutine or context switch information). The maximum size value may be defined at build time.
+
+**Note**: This tool displays static RAM and Flash usage. The heap and stack space are not included in the report as are usually configured at run time, and it's likely to change from one application to another.
 
 There are other entries that require a bit of clarification:
 
 - Fill: represents the bytes in multiple sections (RAM and Flash) that the toolchain has filled with zeros because it requires subsequent data or code to be aligned appropriately in memory.
-- Misc: usually represents helper libraries introduced by the toolchain (like ``libc``), but can also represent modules that are not part of mbed.
+- Misc: usually represents helper libraries introduced by the toolchain but can also represent modules that are not part of mbed.
+
+## Other useful information
+
+It's possible to specify the depth value of the directory level when generating a report. By default *memap* is configured as `--depth=2`, but note that specifying `--depth=0` will display a full report. For example:
+
+```
+$ python mbed-os/tools/memap.py BUILD/K64F/GCC_ARM/example.map -t GCC_ARM --depth 0
+```
+
+As *memap* is automatically invoked when compiling an application with mbed CLI, it's also possible to specify the depth level with `--stats-depth`. For example:
+
+```
+mbed compile -t GCC_ARM -m K64F --stats-depth 3
+```
 
 ## Current support
 
-We have tested *memap* on Windows 7, Linux and Mac OS X. The GCC_ARM (GNU ARM Embedded Toolchain), ARM (ARM Compiler 5) and IAR toolchains generate memory map files.
+We have tested *memap* on Windows 7, Windows 10, Linux and Mac OS X. The GCC_ARM (GNU ARM Embedded Toolchain), ARM (ARM Compiler 5) and IAR toolchains generate memory map files.
 
 ## Known issues and new features
 
-This utility is considered "alpha" quality at the moment. The information generated by this utility may not be fully accurate and may vary from one toolchain to another.
+The information generated by this utility may not be fully accurate and may vary from one toolchain to another. This might be caused by custom definitions of the linker scripts files or by the usage of non standard memory sections.
```

Comment:
For active voice, please change this to "The information this utility generates may not be fully accurate and may vary form one toolchain to another. Custom definitions of the linker script files or usage of nonstandard memory sections may cause this variation." (Please tweak this wording if it changes the meaning.)